gradle.properties

```diff
@@ -12,6 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-version=1.0.0-SNAPSHOT
+version=0.1.0-SNAPSHOT
```

Comment:
I changed this to `0.1.0-SNAPSHOT` as I can plan & test the github action for a `0.1.0` test release.